src/components/Input.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { View, StyleSheet, TextInput } from 'react-native';
+
+export type InputProps = {
+  value: string
+  onChange: (text: string) => void
+  placeholder: string,
+  editable?: boolean,
+  children?: JSX.Element
```

Comment:
This is not explain anything. Input is not a container, so children sounds weird. Icon? Ok. Button? Ok...not children